playground/bicep/BicepSample.AppHost/aspire-manifest.json

```diff
@@ -0,0 +1,181 @@
+        "value": {
+          "type": "string"
+        }
+      }
+    },
+    "test": {
+      "type": "azure.bicep.v0",
+      "path": "test.bicep",
+      "params": {
+        "test": "{val.value}",
+        "values": [
+          "one",
+          "two"
+        ]
+      }
+    },
+    "kv": {
+      "type": "azure.bicep.v0",
+      "connectionString": "{kv.outputs.vaultUri}",
+      "path": "aspire.hosting.azure.bicep.keyvault.bicep",
+      "params": {
+        "principalId": "",
+        "principalType": "",
+        "vaultName": "kv"
+      }
+    },
+    "appConfig": {
+      "type": "azure.bicep.v0",
+      "connectionString": "{appConfig.outputs.appConfigEndpoint}",
+      "path": "aspire.hosting.azure.bicep.appconfig.bicep",
+      "params": {
+        "configName": "appconfig",
+        "principalId": "",
+        "principalType": ""
+      }
+    },
+    "storage": {
+      "type": "azure.bicep.v0",
+      "path": "aspire.hosting.azure.bicep.storage.bicep",
+      "params": {
+        "principalId": "",
+        "principalType": "",
+        "storageName": "storage"
+      }
+    },
+    "blob": {
+      "type": "azure.bicep.v0",
+      "connectionString": "{storage.outputs.blobEndpoint}",
+      "parent": "storage"
+    },
+    "sql": {
+      "type": "azure.bicep.v0",
+      "connectionString": "Server=tcp:{sql.outputs.sqlServerFqdn},1433;Encrypt=True;Authentication=\u0022Active Directory Default\u0022",
+      "path": "aspire.hosting.azure.bicep.sql.bicep",
+      "params": {
+        "principalId": "",
+        "principalName": "",
+        "serverName": "sql",
+        "databases": [
+          "db"
+        ]
+      }
+    },
+    "db": {
+      "type": "azure.bicep.v0",
+      "connectionString": "{sql.connectionString};Intial Catalog=db",
+      "parent": "sql"
+    },
+    "username": {
+      "type": "parameter.v0",
+      "value": "{username.inputs.value}",
+      "inputs": {
+        "value": {
+          "type": "string"
+        }
+      }
+    },
+    "password": {
+      "type": "parameter.v0",
+      "value": "{password.inputs.value}",
+      "inputs": {
+        "value": {
+          "type": "string"
+        }
+      }
+    },
+    "postgres2": {
+      "type": "azure.bicep.v0",
+      "connectionString": "Host={postgres2.outputs.pgfqdn};Username={username.value};Password={password.value}",
+      "path": "aspire.hosting.azure.bicep.postgres.bicep",
+      "params": {
+        "serverName": "postgres2",
+        "administratorLogin": "{username.value}",
+        "administratorLoginPassword": "{password.value}",
+        "databases": [
+          "db2"
+        ]
+      }
+    },
+    "db2": {
+      "type": "azure.bicep.v0",
+      "connectionString": "{postgres2.connectionString};Database=db2",
+      "parent": "postgres2"
+    },
+    "cosmos": {
+      "type": "azure.bicep.v0",
+      "connectionString": "AccountEndpoint={cosmos.outputs.documentEndpoint};AccountKey={keys({cosmos.outputs.accountName})}",
```

Comment:
Is this something that AZD needs to understand so it can go fetch a key from the resource?

Reply:
Yep